### ISSUE TITLE
[iOS] softKeyboard is placed below status bar when connect physical mobile devices to bluetooth keyboard

### ISSUE THREAD

**Issue by JellyHell** [flutter/flutter#56011]
## Flutter environment
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.4 19E287,
    locale zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
[✓] Android Studio (version 3.6)
[✓] Connected device (1 available)

• No issues found!

## Steps to Reproduce

1. connect ios physical mobile devices to bluetooth keyboard
2. input some thing in textInput 
3. device softkeyboard show at the top of  device frame

**Expected results:** 

softkeyboard should show below the input widget，just as it show at Android device

**Actual results:** 
![281588224112_ pic_hd](https://user-images.githubusercontent.com/4350951/80674841-9038b200-8ae5-11ea-8094-c718dbee144f.jpg)

**Comment by TahaTesser** [comment]
Hi @JellyHell 
Testing with bluetooth keyboard, i wasn't able to reproduce this issue
To better address the issue, would be helpful
if you could post a minimal code sample to reproduce the problem
Thank you

**Comment by JellyHell** [comment]
Hi @TahaTesser 


  TextField(
              controller: TextEditingController(),
              style: TextStyle(fontSize: 15, color: AppColors.MainBlackText),
              onChanged: textChanged,
              maxLines: widget.mData.maxLine,
              strutStyle: StrutStyle(height: widget.mData.mStrutHeight),
              decoration: InputDecoration(
                  hasFloatingPlaceholder: false,
                  border: InputBorder.none,
                  contentPadding: EdgeInsets.only(left: 10,top: 12,right: 10,bottom: 6),
                  hintText: widget.mData.mHint,
                  hintStyle: TextStyle(color: AppColors.Color_BE, fontSize: 15)),
            ),
    ),

**Comment by TahaTesser** [comment]
Hi @JellyHell 
Using the bluetooth keyboard on iPhone XR, doesn't show move the layout.

```
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      home: Scaffold(
        appBar: AppBar(
          title: Text('Material App Bar'),
        ),
        body: TextField(
          controller: TextEditingController(),
          style: TextStyle(fontSize: 15, color: Colors.black),
          onChanged: (value) {},
          maxLines: 4,
          strutStyle: StrutStyle(height: 2),
          decoration: InputDecoration(
              hasFloatingPlaceholder: false,
              border: InputBorder.none,
              contentPadding:
                  EdgeInsets.only(left: 10, top: 12, right: 10, bottom: 6),
              hintText: 'Hint Text',
              hintStyle: TextStyle(color: Colors.blue, fontSize: 15)),
        ),
      ),
    );
  }
}

```

Maybe if you could provide complete and proper steps to reproduce the issue from a device
Thank you

**Comment by JellyHell** [comment]
Hi @TahaTesser  Thanks for you response:
1. I create a new flutter project and copy your code 
2.run the project on de physical iOS mobile (iPhone 7  13.3)
3.connect the mobile with bluetooth keyboard 
4.change to input Chinese mode
5.input something

![341588818807_ pic](https://user-images.githubusercontent.com/4350951/81248396-cbd7fc80-904e-11ea-8cfb-c347779bbd01.jpg)

I think this matter to the Chinese input mode . you can change your input mode to Chinese to try.

**Comment by SoolyChristy** [comment]
because FlutterTextInputViewAccessibilityHider frame: x:0, y:0 width: 1, height: 1

**Comment by pedromassangocode** [comment]
Hi @JellyHell 
Are you still having this same issue?

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
